app/models/item.rb

```diff
@@ -0,0 +1,2 @@
+class Item < ApplicationRecord
```

Comment:
We'll need to validate presence of `title`, `price`, `description`, `image` and `category_id`.